### PR TITLE
Bug fix for resuming the training properly

### PR DESCRIPTION
**Pull request by TjarkMiener**
When resuming the training the validation loss of the best performing checkpoint was not track. CTLearn/keras uses the default initial value which is inf. This bug fix allows to resume training properly. Related to [keras issue #12803](https://github.com/keras-team/keras/issues/12803).

### CODE REVIEW

**Review by nietootein — APPROVED** [review]
Go! Thanks for catching this one, @TjarkMiener
